src/lib/share/index.js

```diff
@@ -178,18 +178,31 @@ type ActionType = 'receive' | 'send'
 export function generateShareLink(action: ActionType = 'receive', params: {} = {}): string {
   // depending on the action, routes may vary
   const destination = {
-    receive: 'Send',
-    send: 'Home',
+    receive: Config.receiveUrl,
+    send: Config.sendUrl,
   }[action]
+  let queryParams = ''
+
+  if (params.code) {
```

Comment:
whats the rational for this?
how do we handle the fact that the short links i've created gdlr.info/<code> expects <code> not as a query parameters but as a url section?
How would we handle the regular link for non-production that expects <code> to be a query param?

Reply:
Yes, this is related to the short url, which already has code...but there is no problem for local environment, because the default value also has code now, so it works as expected